jwst/wfs_combine/wfs_combine_step.py

```diff
@@ -37,7 +37,7 @@ def process(self, input_table):
             output_model = wfs.do_all()
             output_model.meta.cal_step.wfs_combine = 'COMPLETE'
             self.save_model(
-                output_model, suffix='wfscmb', output_file=outfile
+                output_model, suffix='wfscmb', output_file=outfile, format=False
```

Comment:
Does the step module need to be explicitly saving the output model, as opposed to just returning the model and letting stpipe handle it?